PusherServer/PusherServer.nuspec

```diff
@@ -1,24 +1,24 @@
 <?xml version="1.0"?>
 <package >
   <metadata>
-    <id>PusherServer</id>
-    <version>$version$</version>
-    <title>Pusher .NET Server Library</title>
-    <authors>Pusher, Phil Leggetter, John McLoughlin</authors>
+    <id>PusherServerInnov8tive</id>
+    <version>4.0.5</version>
```

Comment:
This shouldn't be hardcoded, we should be using the tokenised version that was here previously